projects/plugins/jetpack/tests/php/general/test_class.jetpack.php

```diff
@@ -398,7 +398,7 @@ function test_is_dev_version_false_with_number_dot_number_dot_number() {
 	/**
 	 * Tests is_offline_mode filter.
 	 *
-	 * @covers \Automattic\Jetpack\Status::is_offline_mode
+	 * @covers \Automattic\Jetpack\Status
```

Comment:
Perhaps these two tests should be moved to projects/packages/status/tests/php/test-status.php?